.pre-commit-config.yaml

```diff
@@ -9,25 +9,12 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
 
-  - repo: https://github.com/asottile/pyupgrade
-    rev: "v3.3.1"
-    hooks:
-      - id: pyupgrade
-        args: ["--py37-plus"]
-
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: "v0.0.228"
+    rev: "v0.0.230"
     hooks:
       - id: ruff
-        # Respect `exclude` and `extend-exclude` settings.
-        args: ["--force-exclude"]
```

Comment:
`--force-exclude` is now [included by default](https://github.com/charliermarsh/ruff-pre-commit/blob/v0.0.230/.pre-commit-hooks.yaml#L4) in the pre-commit configuration, so we can remove it.